monash-human-power/dashboard
# Move prettier config file to root directory

## Description



Did a clean clone of the repository. VS code was incorrectly formatting the js file. By moving the `.prettierrc` file that was already in the `/server` directory, into the root directory, I was able to get vs code to autoformat the JS code using the prettier formatter.

## Screenshots



N/A

## Steps to Test



Make some changes in a file, save file whilst in vs code. VS code should format the document automatically.
